### PR TITLE
REP 227 add persona tests for incident reporting

### PR DESCRIPTION
**Pull request by TylerOSI**
This adds a persona test for incident reporting

### CODE REVIEW

**Comment by TylerOSI** [comment]
just noticed the code climate issues. ill get on that

**Comment by ratneshraval** [comment]
@TylerOSI Approved but there are 2 code climate issues. Can you fix them before merging.

File: spec/page_objects/incident-report/reporter_template.rb [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
```
Comment (tparkerosi) [inline spec/page_objects/incident-report/reporter_template.rb]: This file isn't following the pattern of the other Page Objects. Not sure if the word 'template' adds anything useful. I'm not sure what template means in this context.

Comment (TylerOSI) [reply spec/page_objects/incident-report/reporter_template.rb]: yeah there may be a better word to describe the page where you enter the reporter's information.

**Pull request merged**